File: speed-loop.c
#include <stdio.h>
#include <time.h>

int main() {
  int cycles = 1000000000;
  clock_t start, end;
  double duration;

  start = clock();
  
  for (int i = 0; i < cycles; i++){
    // Empty loop
  }

  end = clock();
  duration = ((double) (end - start) / CLOCKS_PER_SEC);
  printf("C looped %d times in %1f seconds \n", cycles, duration);

  return 0;
}

/* To compile
gcc speed-loop.c -o speed-loop

To run ./speed-loop
*/

File: speed-loop.py
import time

def main():
  cycles = 1000000000
  start = time.perf_counter()

  for i in range(0, cycles):
    pass # This is an empty loop

  end = time.perf_counter()

  duration = end - start

  print("Python 3 looped %s time in %s seconds" % (cycles, duration))

main()